src/Drupal/DrupalExtension/Context/MessageContext.php

```diff
@@ -361,4 +361,55 @@ private function assertNot($message, $selectorId, $exceptionMsg)
             }
         }
     }
+
+  /**
+   * Check if the current page contains no messages of the specified selector.
+   *
+   * @Then /^I should see no (?P<selector>(?:success|error|warning)) message$/
+   */
+  public function assertSelectorNotVisible($selector):void {
+    if ($selector == 'success') {
+      $selector_id = 'success_message_selector';
+    }
+    else if (selector == 'error') {
+      $selector_id = 'error_message_selector';
+    }
+    else {
+      $selector_id = 'warning_message_selector';
+    }
+    $this->assertNotVisible($selector_id);
```

Comment:
@kumikoono This should be:

```suggestion
  public function assertSelectorNotVisible($selector): void {

    $selector_id = 'warning_message_selector';

    if ($selector == 'success') {
      $selector_id = 'success_message_selector';
    }

    if (selector == 'error') {
      $selector_id = 'error_message_selector';
    }

    $this->assertNotVisible($selector_id);
```

Reply:
Changed to use switch.